charts/apiclarity/templates/deployment.yaml

```diff
@@ -76,11 +76,13 @@ spec:
               value: {{ .Values.apiclarity.notificationURLPrefix }}
             {{- end }}
             - name: FUZZER_DEPLOYMENT_TYPE
-              value: kubernetes
+              value: {{ .Values.apiclarity.env.plugins.FUZZER_DEPLOYMENT_TYPE }}
```

Comment:
Also, you need to create `FUZZER_DEPLOYMENT_TYPE` in the values.yaml file https://github.com/openclarity/apiclarity/blob/devel-global-specs/charts/apiclarity/values.yaml#L53

Reply:
already done. was needed by CI / Lint Helm Chart